.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           version: nightly
 
       - name: Run tests
-        run: forge test --no-match-path "test/spoke/fuzzing/**/*.sol,test/integration/fork/**/*.sol" --deny-warnings
+        run: forge test --no-match-path "test/integration/fork/**/*.sol" --deny-warnings
```

Comment:
This should have worked but fine with merging. I'll fix it when we finally merge the invariant tests 😅